lib/jsdom/browser/Window.js

```diff
@@ -168,70 +171,40 @@ function Window(options) {
     return result;
   };
 
-  this.XMLHttpRequest = function () {
-    var xhr = new XMLHttpRequest();
-    var lastUrl = "";
-    xhr._open = xhr.open;
-    xhr.open = function (method, url, async, user, password) {
-      lastUrl = fixUrlForBuggyXhr(resolveHref(window.document.URL, url));
-      return xhr._open(method, lastUrl, async, user, password);
-    };
-    xhr._getAllResponseHeaders = xhr.getAllResponseHeaders;
-    xhr.getAllResponseHeaders = function () {
-      if (lastUrl.startsWith("file:")) {
-        // Monkey patch this function for files. The node-xhr module will crash for file URLs.
-        return null;
-      }
-      return xhr._getAllResponseHeaders();
-    };
-    xhr._send = xhr.send;
-    xhr.send = function (data) {
-      var cookieJar = window.document._cookieJar;
-      var cookieStr = cookieJar.getCookieStringSync(lastUrl, {http: true});
-
-      if (cookieStr) {
-        xhr.setDisableHeaderCheck(true);
-        xhr.setRequestHeader("cookie", cookieStr);
-        xhr.setDisableHeaderCheck(false);
-      }
+  this.File = file.File;
```

Comment:
As per the above comment, the main export of `file.js` should be a function like

```js
module.exports = core => {
  core.File = File;
  core.Blob = Blob;
};
```

Then no code needs to appear here in window.js at all.